test/goldens/end2end/angular_components-0.10.0.json

```diff
@@ -45991,6 +45991,12 @@
         "grantedPoints": 20,
         "maxPoints": 20,
         "summary": "Package gets 20 points if its Dart sdk constraint allows Dart 2."
+      },
+      {
+        "title": "Packaging conventions",
+        "grantedPoints": 15,
+        "maxPoints": 20,
+        "summary": "Package gets 10 points for a valid pubspec.yaml and 5 points for each of README.md and CHANGELOG.md.\n\nLinks in README.md should be secure. 1 link is insecure.\n`README.md:51:22`\n\n```\n   ╷\n51 │ component's provided <a href=\"http://sass-lang.com/guide#topic-6\">Sass mixins</a>.</p>\n   │                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n   ╵\n```\n\nUse `https` URLs instead."
```

Comment:
I'm wondering if it is important to provide the full context of the link, maybe the link itself would be enough?

Reply:
But isn't it nice to get it with context?

Reply:
Yeah, let's keep it and fix only when there is a problem with that.